cmd/agent/common/helpers.go

```diff
@@ -49,3 +52,46 @@ func setupConfig(confFilePath string, configName string, withoutSecrets bool) er
 	}
 	return nil
 }
+
+// SetupSystemProbeConfig reads the system-probe.yaml into the global config object
+func SetupSystemProbeConfig(sysProbeConfFilePath string) error {
+	var (
+		file *os.File
+		err  error
+	)
+
+	// Open the system-probe.yaml file if it's in a custom location
+	if len(sysProbeConfFilePath) != 0 {
+		// If file is set directly,  assume we should use it
+		if strings.HasSuffix(sysProbeConfFilePath, ".yaml") {
+			// Open the file if they pass it in directly
+			file, err = os.Open(sysProbeConfFilePath)
+		} else {
+			file, err = os.Open(sysProbeConfFilePath + "/system-probe.yaml")
+		}
+	} else {
+		// Assume it is in the default location if nothing is passed in
+		file, err = os.Open(DefaultConfPath + "/system-probe.yaml")
+	}
+
+	defer file.Close()
+	if err != nil {
+		return err
+	}
+
+	// Merge config with an IO reader since this lets us merge the configs without changing
+	// the config file set with viper
+	if err := config.Datadog.MergeConfig(file); err != nil {
```

Comment:
My concern here is that the systes-probe.yaml and the datadog.yaml will have different default/value and we will overwrite the one in datadog.yaml here. I think it would be very confusing for both customers and support.

My understanding is that we only need 1 value in that file, why not just use `yaml.Load` to get that value and set it by hand in the configuration ?